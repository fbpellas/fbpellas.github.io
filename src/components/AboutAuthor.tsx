import * as React from 'react';
import { AUTHOR_FIRSTNAME, AUTHOR_FULLNAME, BASE_PATH_IMG } from '../constants';

export interface AboutAuthorProps { }

const AboutAuthor: React.FC<AboutAuthorProps> = (_props) => {
  return <div className="block-2">
    <div className="article">
      <h3 className="h3-title">About the Author</h3>
      <div className="flex-wrapper">
        <div className="flex-1">
          <img className="full-img" src={`${BASE_PATH_IMG}faith.jpg`} alt={AUTHOR_FULLNAME} />
        </div>
        <div className="author-text flex-2">
          <p>
            {AUTHOR_FULLNAME} is a scholar at the University of San Francisco’s TESOL department. For the past
            four years, she has been teaching English to learners from beginners to advanced levels. When she’s
            not working on her thesis, {AUTHOR_FIRSTNAME} loves learning French, watercolor painting, and
            sending postcards to her nearest and dearest.
          </p>
        </div>
      </div>
    </div>
  </div>
};

export { AboutAuthor };