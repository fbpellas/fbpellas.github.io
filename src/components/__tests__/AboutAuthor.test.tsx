import React from 'react';
import { render } from '@testing-library/react';
import { AboutAuthor } from '../AboutAuthor';

test('AboutAuthor', () => {
  const { container } = render(<AboutAuthor />);
  expect(container).toMatchSnapshot();
});
