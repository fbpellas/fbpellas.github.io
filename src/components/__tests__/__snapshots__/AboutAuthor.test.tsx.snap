// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutAuthor 1`] = `
<div>
  <div
    class="block-2"
  >
    <div
      class="article"
    >
      <h3
        class="h3-title"
      >
        About the Author
      </h3>
      <div
        class="flex-wrapper"
      >
        <div
          class="flex-1"
        >
          <img
            alt="Faith Pellas"
            class="full-img"
            src="https://raw.githubusercontent.com/fbpellas/fbpellas.github.io/main/public/img/faith.jpg"
          />
        </div>
        <div
          class="author-text flex-2"
        >
          <p>
            Faith Pellas
             is a scholar at the University of San Francisco’s TESOL department. For the past four years, she has been teaching English to learners from beginners to advanced levels. When she’s not working on her thesis, 
            Faith
             loves learning French, watercolor painting, and sending postcards to her nearest and dearest.
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
